gamestonk_terminal/cryptocurrency/discovery/discovery_controller.py

```diff
@@ -0,0 +1,232 @@
+"""Cryptocurrency Overview Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302, W0622
+import argparse
+import os
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.discovery import (
+    pycoingecko_view,
+    coinpaprika_view,
+    coinmarketcap_view,
+    binance_view,
+)
+
+
+class Controller:
+
+    CHOICES = [
+        "?",
+        "cls",
+        "help",
+        "q",
+        "quit",
+        "cp_search",
+        "cp_coins",
+        "cmc_top",
+        "cg_trending",
+        "cg_most_voted",
+        "cg_most_visited",
+        "cg_top_volume",
+        "cg_recently",
+        "cg_sentiment",
+        "cg_gainers",
+        "cg_losers",
+        "cg_yfarms",
+        "cg_top_defi",
+        "cg_top_dex",
+        "cg_top_nft",
+        "cg_coins",
+        "bin_coins",
+    ]
+
+    def __init__(self):
+        """CONSTRUCTOR"""
+
+        self._discovery_parser = argparse.ArgumentParser(add_help=False, prog="disc")
+        self._discovery_parser.add_argument("cmd", choices=self.CHOICES)
+
+    def print_help(self):
+        """Print help"""
+        print("\nDiscovery:")
```

Comment:
One of the new refactoring rules we'll implement is to build the whole string with """ and then print(help_text) at the end. Not sure if you want to do this now or later